toolkit/imageconfigs/packagelists/core-tools-packages.json

```diff
@@ -1,7 +1,6 @@
 {
     "packages": [
         "dnf",
-        "vim",
```

Comment:
kinda think we should go the other way on this and define a marketplace-tools-packages.json and only use that.   Otherwise nearly every other image type will be impacted.

Reply:
Thank you Jon. Done the change to only remove vim from marketplace images.